docs/tutorials/index-to-local-csv-files.md

```diff
@@ -0,0 +1,217 @@
+
+```bash
+sqd init local-csv-indexing -t evm
+```
+
+Where `local-csv-indexing` is the name of the project, and can be changed to anything else. And `-t evm` specifies what template should be used and makes sure the setup will create a project from the EVM-indexing template.
+
+
+:::info
+**Note:** The template actually has more than what we need for this project. In the repository used for this article, unnecessary packages have been removed, so the [repository's `package.json`](https://github.com/RaekwonIII/local-csv-indexing/blob/main/package.json) can be used.
+
+Similarly, the [list of available commands](https://github.com/RaekwonIII/local-csv-indexing/blob/main/commands.json) has been shortened, `docker-compoes.yml` has been removed, as well as `schema.graphql`, and `squid.yaml`, because they will not be used.
+:::
+
+### ERC-20 token ABI
+
+To be able to index the transfers of a token, it's necessary to have the token's address, as well as the smart contract's ABI (Abstract Binary Interface), which defines the contract's functions, the events, as well as their inputs and outputs, and their types.
+
+Luckily, both of these can be found on block explorers like [Etherscan](https://medium.com/r/?url=https%3A%2F%2Fetherscan.io%2Ftoken%2F0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0).
+
+The indexer needs the ABI, because that's how it can decode the contract's events or functions inputs (and outputs), and provide the information they carry. The SDK has a handy command to generate some boilerplate TypeScript code to achieve this.
+
+```bash
+sqd typegen 0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0#matic
+```
+:::info
+Because the contract is deployed on Ethereum, and available through Etherscan API, the SDK is able to generate the necessary code with just the smart contract address.
+
+For all other cases, the same command needs the path to a JSON file containing the smart contract's ABI, instead of its address.
+:::
+
+This will generate some files under the `src/abi` folder, the most interesting of which is `matic.ts`.
+
+### CSV, Tables and Databases
+
+It's necessary to install the `file-store-csv` package, because the goal is to save  data on CSV files locally, and this package is not available by default:
+
+```bash
+npm i @subsquid/file-store-csv
+```
+
+:::info
+Equivalent packages for parquet files, and for uploading files using S3-compatible APIs are also available
+:::
+
+Next, let's created a new file, named `tables.ts`, in case the project should be expanded to multiple files. This is where it's possible to provide a filename for the CSV files, as well as configure their data structure, the same way as if they were a database table (the class name is no coincidence):
+
+```typescript
+import {Table, Column, Types} from '@subsquid/file-store-csv'
+
+export const Transfers = new Table(
+    'transfers.csv',
+    {
+        blockNumber: Column(Types.Integer()),
+        timestamp: Column(Types.DateTime()),
```

Comment:
Let's rename this type to `Timestamp`

Reply:
What do you mean? I can rename the field...the type is defined in the library...

Reply:
Package updated

Reply:
Is it available on npm? Cause I don't see it
https://www.npmjs.com/package/@subsquid/file-store-csv